stories/molecules/components/header/Header.stories.mdx

```diff
@@ -27,27 +32,65 @@ import './header.scss'
   ]}
 />
 
-# Header component
+# Header
+
+> Provides a consistent user experience and helps users navigate the site.
 
 [![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
 
 ## Description
 
-**Header component** is a responsive component which embeds two main elements:
+**Header** is a responsive component which embeds two main elements:
 
-- A `Logo` which must be part of the set of applications representing the ØKP4 brand.
+- A `firstElement` that positions itself first in the reading flow. Typically used for logos. Will resize if larger than max size.
 - A `ThemeSwitcher` which allows each application to automatically manage the themes offered by the ØKP4 design system (Blue Stone / Light Stone).
 
 ## Overview
 
-<Story name="Overview">
-  <div style={{ width: '100%' }}>
-    <Header />
-  </div>
+<Story
+  name="Overview"
+  argTypes={{
+    firstElement: {
+      control: false,
+      table: {
+        type: { summary: 'JSX.Element' }
+      }
+    }
+  }}
+>
+  <Header firstElement={renderLogo()} />
 </Story>
 
+export const renderLogo = () => {
+  const { isXSmall } = useBreakpoint()
```

Comment:
This must be handled by the component itself, it should not be the responsibility of the parent to add a media query regarding the size of the viewport.